compiler/src/iree/compiler/Codegen/LLVMGPU/test/vector_lowering.mlir

```diff
@@ -126,11 +121,7 @@ func.func @multi_reduction_no_uplift(%a: vector<2x1x8xf32>, %b: vector<2x1x8xf32
 }
 
 // CHECK-LABEL: func.func @multi_reduction_no_uplift
-// CHECK-DAG:  %[[C0:.+]]  = arith.constant 0.000000e+00 : f32
-// CHECK-DAG:  %[[V0:.+]]  = arith.constant dense<0.000000e+00> : vector<2x1x8xf32>
-// CHECK:      %[[MUL:.+]] = arith.mulf %{{.*}}, %{{.*}} fastmath<fast> : vector<2x1x8xf32>
-// CHECK:      %[[ADD:.+]] = arith.addf %[[MUL]], %[[V0]] : vector<2x1x8xf32>
-// CHECK:      %[[E0:.+]]  = vector.extract %[[ADD]][0, 0] : vector<8xf32> from vector<2x1x8xf32>
-// CHECK:                    vector.reduction <add>, %[[E0]], %[[C0]] : vector<8xf32> into f32
-// CHECK:      %[[E1:.+]]  = vector.extract %[[ADD]][1, 0] : vector<8xf32> from vector<2x1x8xf32>
-// CHECK:                    vector.reduction <add>, %[[E1]], %[[C0]] : vector<8xf32> into f32
+// CHECK-NOT: math.fma
+// CHECK-COUNT-2: arith.mulf
+// CHECK-COUNT-2: arith.addf
+// CHECK-COUNT-2: vector.reduction
```

Comment:
It looks like these tests have some fastmath checks, can we keep those in?

Reply:
Also can you do a basic test for the insert/extracts generated when unrolling. Can be a part of a different test that looks like
```
func.func @elementwise_unroll(%arg0: vector<3x4xf32>, %arg1: vector<3x4xf32>) -> vector<3x4xf32> {
  %0 = arith.addf %arg0, %arg1 : vector<3x4xf32>
  return %0 : vector<3x4xf32>
}
```
or can be part of this test